### PR TITLE
Task nav

### PR DESCRIPTION
**Pull request by paulms1**
Finished nav/footer/hero banner

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
```
Comment (ChrisTregaskis) [inline index.html]: missing the lang="en-gb"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	<head>
+		<title>Paul's portfolio</title>
+		<link rel="stylesheet" type="text/css" href="styles.css">
+		<link href="https://fonts.googleapis.com/css?family=Oswald&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
```
Comment (ChrisTregaskis) [inline index.html]: indentation on the link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		<meta name="viewport" content="width=device-width">
+		</head>
+	<body>
+    	<nav>
```
Comment (ChrisTregaskis) [inline index.html]: indentation of nav

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	<head>
+		<title>Paul's portfolio</title>
+		<link rel="stylesheet" type="text/css" href="styles.css">
+		<link href="https://fonts.googleapis.com/css?family=Oswald&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+		<meta name="viewport" content="width=device-width">
+		</head>
+	<body>
+    	<nav>		
+		   <div class="logo"><img src="images/logo.png"></div>
```
Comment (ChrisTregaskis) [inline index.html]: img missing alt="what the image is"

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+ 			 <div class="categories"> 
+			   <ul>
+			   	 <li><a href="#">ABOUT ME</a></li>
+				 <li><a href="#">PORTFOLIO</a></li>
+				 <li><a href="#">CONTACT</a></li>
+			   </ul>	
+			 </div> 
+		</nav>
+		<div class="hero">
+			<img src="images/hero.jpg">
```
Comment (ChrisTregaskis) [inline index.html]: alt=" " missing

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+}
+
+.job {
+	position: absolute;
+	top: 200px;
+	color: RGB(232,85,58);
+	font-size: 2.5rem;
+}
+
+.tagLineOne{
```
Comment (ChrisTregaskis) [inline styles.css]: indent {

**Comment by paulms1** [comment]
About me task:

-  Headshot image
- Blurb

Very simple - MVP in mind

File: index.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+		<section class="aboutMe">
+			<div class="headShotContainer">
+				<div class="headShot">
+					<img src="images/myFace.jpg" alt="paul's face">
+				</div>
+			 </div>
+			<h2>ABOUT ME</h2>
+			<p>I am a trainee fullstack developer currently studying at Mayden Academy and loving every minute of it.</p>
+			<p>My background is in language teaching and I am a Spanish speaker. I was first drawn to coding when I had the idea for a language learning app for my students - something which I aspire to make further down the road on my journey to becoming an accomplished developer.</p>
+		</section>
```
Comment (markandmarkup) [inline index.html]: All looks good here!

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,401 @@
...
+}
+
+
+.aboutMe h2 {
+	font-family: 'Oswald', sans-serif;
+	margin-top: 15px;
+	padding: 0;
+}
+.aboutMe p {
+	padding-left: 0 4% 0 4%;
```
Comment (markandmarkup) [inline styles.css]: Padding-left only takes one value. The shorthand "padding" can take four

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,401 @@
...
+	margin: 0 0 10px 5px;
+	/*color: RGB(232,85,58);*/
```
Comment (markandmarkup) [inline styles.css]: You should remove any commented out lines

**Pull request merged**